### PR TITLE
Add project: Chef Cookbooks and Tests with Kitchen and ServerSpec

### PR DESCRIPTION
**Pull request by allyshoww**

### CODE REVIEW

**Comment by diegows** [comment]
Maybe I'm missing something, but I do not see where is the code that's installing OpenVPN here. Also, "kitchen verify' should be enough to test, we don't run rspec or other commands to test usually.

Take a look at the example commands at https://kitchen.ci/

**Comment by allyshoww** [comment]
@diegows The code that installing openvpn is:
image

Sure, thanks for tips. I was updated the code with "kitchen verify"

**Review by allyshoww — COMMENTED** [review]
@diegows This is the PR:

We have the manual test with kitchen and rspec and automatic test with kitchen verify (Makes a lot of sense)

File: openvpn/installpkg/recipes/Rakefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+require 'rake'
```
Comment (diegows) [inline openvpn/installpkg/recipes/Rakefile]: What is this file doing exactly?

File: openvpn/installpkg/recipes/copy_file.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+cookbook_file "Copy a file" do
```
Comment (diegows) [inline openvpn/installpkg/recipes/copy_file.rb]: Why do we need this?

Comment (allyshoww) [reply openvpn/installpkg/recipes/copy_file.rb]: I use this cookbook to upload this file for use in serverspec-init. But, with "kitchen verify", I see this is unnecessary. I will delete these files and update "README.md"

File: openvpn/installpkg/recipes/Rakefile (hunk 1 above)
Comment (allyshoww) [reply openvpn/installpkg/recipes/Rakefile]: It's to use with serverspec-init for a manual test. With "kitchen verify", I see this is unnecessary and will be deleted.

**Review by diegows — APPROVED** [review]
Looks much better now. Approved. Merge please.

**Pull request merged**